src/MPoly.jl

```diff
@@ -448,7 +448,10 @@ end
 function content(a::MPolyRingElem{T}) where T <: RingElement
    z = zero(coefficient_ring(a))
    for c in coefficients(a)
-      z = gcd(z, c)
+      z = gcd!(z, c)
+      if isone(z)
+         break
+      end
```

Comment:
This is "wrong", since to make things "consistent", we need the GCD of the polynomial with coefficient `1, 1, 1//2` to be `1//2`.

Reply:
ahh, true... This is basically the same reason as https://github.com/Nemocas/AbstractAlgebra.jl/pull/1971.